.gitignore

```diff
@@ -8,7 +8,11 @@ Include
 bin
 gen
 obj
-libs
+**/libs/eegeo/
```

Comment:
Please can you explain the rationale behind moving the existing libs in libs/eegeo? Could we consider making a subdirectory for the newly provided external libs (eg. libs/external or libs/cardboard) as a less invasive alternative?